src/constants.js

```diff
@@ -33,6 +33,9 @@ export const REDUCERS = deepFreeze(['worker', 'socket', 'plugin', 'polling']);
 
 export const DEFAULT_POLLING_INTERVAL = 30 * 1000;
 
+export const SOUND_URL = 'soundUrl';
```

Comment:
I was thinking of maybe calling this `notificationSoundUrl` instead of just `soundUrl` since that's a bit ambiguous. But I'm fine either way.